src/Nethermind/Nethermind.Runner.Test/ConfigFilesTests.cs

```diff
@@ -57,6 +66,83 @@ public void Required_config_files_exist(string configWildcard)
             }
         }
 
+        [TestCase(typeof(IInitConfig))]
+        [TestCase(typeof(IBaselineConfig))]
+        [TestCase(typeof(ISyncConfig))]
+        [TestCase(typeof(IAuraConfig))]
+        [TestCase(typeof(IMiningConfig))]
+        [TestCase(typeof(INdmConfig))]
+        [TestCase(typeof(INdmMongoConfig))]
+        [TestCase(typeof(IDbConfig))]
+        [TestCase(typeof(IBloomConfig))]
+        [TestCase(typeof(IEthStatsConfig))]
+        [TestCase(typeof(IGrpcConfig))]
+        [TestCase(typeof(IJsonRpcConfig))]
+        [TestCase(typeof(IKeyStoreConfig))]
+        [TestCase(typeof(IMetricsConfig))]
+        [TestCase(typeof(IStatsConfig))]
+        [TestCase(typeof(IDiscoveryConfig))]
+        [TestCase(typeof(INetworkConfig))]
+        [TestCase(typeof(IKafkaConfig))]
+        [TestCase(typeof(IAnalyticsConfig))]
+        [TestCase(typeof(IHiveConfig))]
+        [TestCase(typeof(ISeqConfig))]
+        [TestCase(typeof(ITxPoolConfig))]
+        [TestCase(typeof(IVaultConfig))]
+        [TestCase(typeof(IWalletConfig))]
+        public void All_default_values_are_correct(Type configType)
+        {
```

Comment:
the config types should be detected automatically via Assembly GetTypes where type isassignable from IConfig